### PR TITLE
Update Validator Dependency

### PR DESCRIPTION
**Pull request by radamson**
Updates the validator dependency to the most recent version.

### CODE REVIEW

**Comment by radamson** [comment]
Team talked today and decided to just require Java 11.

There are still issues with the [HL7 Library](https://github.com/hapifhir/org.hl7.fhir.core/pull/622) that need to be sorted out, but requiring Java 11 seems reasonable anyways.

File: build.gradle.kts [hunk 1]
```diff
@@ -18,7 +18,7 @@ dependencies {
     // https://chat.fhir.org/#narrow/stream/179166-implementers/topic/New.20validator.20JAR.20location
     // the ig-publisher uses this one too
     // https://github.com/HL7/fhir-ig-publisher/blob/master/pom.xml#L68
-    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.3.14")
+    implementation("ca.uhn.hapi.fhir", "org.hl7.fhir.validation", "5.5.10")
```
Comment (Jack-Fraser16) [inline build.gradle.kts:21]: Latest validation version is now "5.6.21". I tested it locally and it didn't break anything.

Comment (yunwwang) [reply build.gradle.kts:21]: This will be fixed in PR #43 So I think this is fine.

**Review by yunwwang — DISMISSED** [review]
Tested with US Core v3.1.1. Looks good

**Review by Jack-Fraser16 — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (Jack-Fraser16) [reply build.gradle.kts:21]: Sounds good. I'll approve and merge.

**Review by Jack-Fraser16 — APPROVED** [review]

**Review by yunwwang — APPROVED** [review]

**Pull request merged**